### PR TITLE
zf 0.7.0

### PR DESCRIPTION
**Pull request by natecraddock**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Just noting that zf now has a git submodule (I understand this is worse for caching), and whenever Zig supports a built-in package manager the submodule will be removed :)

### CODE REVIEW

File: Formula/zf.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Zf < Formula
   desc "Command-line fuzzy finder that prioritizes matches on filenames"
   homepage "https://github.com/natecraddock/zf"
-  url "https://github.com/natecraddock/zf/archive/refs/tags/0.6.0.tar.gz"
-  sha256 "9767ce1142933c753b749630229d10519c4c0b208f63931e43495acc632135e2"
+  url "https://github.com/natecraddock/zf.git",
```
Comment (chenrui333) [inline Formula/zf.rb:4]: can you add a code comment about when we can revert back to the tarball file? Thanks!

Comment (natecraddock) [reply Formula/zf.rb:7]: Absolutely! I just updated the PR

Comment (chenrui333) [reply Formula/zf.rb:7]: Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @natecraddock!

**Pull request closed without merging**